### ISSUE TITLE
PTQ is faster than QAT

### ISSUE THREAD

**Issue by pangr** [NVIDIA/TensorRT#2204]
## Environment

**TensorRT Version**: 8.4.1.5 
**NVIDIA GPU**:  1080ti
**NVIDIA Driver Version**: 450 
**CUDA Version**: 11.0
**CUDNN Version**: 8.1.0
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Steps To Reproduce

When I use PTQ, 'PWN(PWN(Sigmoid, Mul), Add)' is 'Int8' -> 'Int8':
![image](https://user-images.githubusercontent.com/5838111/182324360-c6370a62-867c-4eb7-99b6-9d31b5321304.png)
But when I use QAT, 'PWN(PWN(Sigmoid, Mul), Add)' is 'Int8' -> 'Float32':
![image](https://user-images.githubusercontent.com/5838111/182324628-eee5f340-3efb-464f-b88d-669323839080.png)

Int8 onnx is:
![image](https://user-images.githubusercontent.com/5838111/182324767-52474344-212c-4f09-b907-151391fd4152.png)

**Comment by zerollzeng** [comment]
@ttyio Do you have any recommendations on the QDQ placement here? I think the user can fine-tune it to get better performance.

**Comment by ttyio** [comment]
@pangr , what's the op after the `add`, also have you tried insert Q/DQ after the `add`? thanks

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
